### DISCUSSION TITLE
AltGr in ISO keyboard layouts

### DISCUSSION THREAD

**Discussion by le-mich** [neovide/neovide#678] [Q&A] [answered]
Hi all,
I'm an italian user and have the infamous ``AltGr`` issue with some characters which makes Neovide unusable for me.

I was wondering what's the status on this issue and if there's something that can be done to solve this.

**Comment by Kethku** [comment, marked as answer]
Status is captured here: https://github.com/Kethku/neovide/issues/445 we are waiting on upstream PRs to merge

**Reply by le-mich** [reply]
Perfect! Thanks for the link, and for your work on Neovide, I can't wait to use it!
